src/components/clients/ClientForm.js

```diff
@@ -18,6 +27,28 @@ class ClientForm extends React.Component {
     });
   }
 
+  onRedirectUrlChange(e) {
+    const focus = this.state[e.target.name + 'Focus'];
+    let urls = Object.assign([], this.state[e.target.name]);
```

Comment:
`Object.assign` on an array, this is not usual :)

I'd rather use `let urls = this.state[e.target.name].slice()` if you want a copy here.